interfaces/builtin/account_control.go

```diff
@@ -72,6 +72,8 @@ func init() {
 		name:                  "account-control",
 		summary:               accountControlSummary,
 		description:           accountControlDescription,
+		implicitOnCore:        true,
```

Comment:
There are things that are implicit on classic, yes. I changed this from the previous code to make it less magic and more obvious as to what is added. I don't mind a single implicit bit-flag but I'd rather do that in a separate branch to allow this one to land and iterate.